src/components/Navbar.js

```diff
@@ -3,91 +3,81 @@ import Notification, { notify } from 'react-notify-toast';
 import { withRouter, Link } from 'react-router-dom';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
+import {
+  Collapse,
+  Navbar as Navigation,
+  NavbarToggler,
+  NavbarBrand,
+  Nav,
+  NavItem,
+  UncontrolledDropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem,
+} from 'reactstrap';
 import { logoutAction } from '../redux/actions/auth/authActions';
 
 export class Navbar extends Component {
+    state = {
+      isOpen: false,
+    };
+
   logoutUser = (event) => {
     event.preventDefault();
     this.props.logoutAction();
     this.props.history.push('/');
     notify.show('You Loggedout successfully', 'success', 3000);
   };
 
+  toggle = () => {
+    this.setState(prevState => ({
+      isOpen: !prevState.isOpen,
+    }));
+  }
+
   render() {
+    const { isOpen } = this.state;
+    const { loggedIn } = this.props;
     return (
       <div>
         <Notification options={{ zIndex: 10000, top: '200px' }} />
-        <nav className="navbar navbar-expand-lg navbar-light bg-dark">
-          <a className="nav-link navbar-brand" href="/">
-              OCL for OpenMRS
-          </a>
-          <button
-            className="navbar-toggler"
-            type="button"
-            data-toggle="collapse"
-            data-target="#navbarA"
-            aria-controls="navbarNavDropdown"
-            aria-expanded="false"
-            aria-label="Toggle navigation"
-          >
-            <span className="navbar-toggler-icon" />
-          </button>
-          <div className="collapse navbar-collapse " id="navbarA">
-            {this.props.loggedIn && (
-              <ul className="navbar-nav ml-auto" id="navList">
-                <li className="nav-item nav-link">
-                  <Link
-                    className="nav-link text-white"
-                    to="/home"
-                  >
-                      Home
-                  </Link>
-                </li>
-                <li className="nav-item nav-link">
-                  <Link
-                    className="nav-link text-white"
-                    to="/dashboard/dictionaries"
-                  >
-                      All Dictionaries
-                  </Link>
-                </li>
-                <li className="nav-item nav-link dropdown">
-                  <a
-                    className="nav-link dropdown-toggle text-white"
-                    href="!#"
-                    id="navbarDropdown"
-                    role="button"
-                    data-toggle="dropdown"
-                    aria-haspopup="true"
-                    aria-expanded="false"
-                  >
+        <Navigation color="light" light expand="md">
+          <NavbarBrand href="/">OCL for OpenMRS</NavbarBrand>
+          <NavbarToggler onClick={this.toggle} />
+          {loggedIn ? (
```

Comment:
Use `&&` here instead of `?`.

Reply:
@dessHub I think the ternary makes more sense here. Any particular reason you want && there?

Reply:
@AGMETEOR I think that in this case using short circuit operator(`&&`) is handier than ternary operator(`?`).

Check this [here](https://reactjs.org/docs/conditional-rendering.html)

Reply:
Alright. Let me use the short circuit.

Reply:
@dessHub I have made the changes.

Reply:
As @muhozi said `&&` make sense since, you just checking for one condition. `?` is suitable for `if ...else` statements.